### PR TITLE
KOGITO-8168 Update Quarkus job: Centralize PR links

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-8168

After this merge, there will be:
- only on message on Zulip with all PRs in there
- each PR is updated with all the related PRs
- default reviewers are assigned

- https://github.com/kiegroup/kogito-pipelines/pull/661
- https://github.com/kiegroup/optaplanner/pull/2264

### CODE REVIEW

**Review by mbiarnes — APPROVED** [review]

File: .ci/jenkins/Jenkinsfile.ecosystem.update-quarkus-all [hunk 1]
```diff
@@ -88,11 +90,10 @@ pipeline {
                                     "${SEED_BRANCH}" != "${SEED_BRANCH_CONFIG_FILE_GIT_BRANCH}") {
                                 String prLink = githubscm.createPR(commitMsg, 'Please review and merge', "${SEED_BRANCH}", "${SEED_AUTHOR_CREDS_ID}")
                                 echo "Created PR ${prLink}"
-
-                                sendNotification("PR was created to update quarkus version in seed repo to ${getNewVersion()}.\nPlease review and merge ${prLink}")
+                                prLinks.add(prLink)
                             } else {
                                 echo 'PR was already created. Will not try to create a new one ...'
-                            }
+                                    }
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.ecosystem.update-quarkus-all]:
```suggestion
                            }
```

File: .ci/jenkins/Jenkinsfile.ecosystem.update-quarkus-all [hunk 2]
```diff
@@ -183,4 +247,26 @@ void getOrCreateGitBranch(String branch, String credentialsId) {
...
+}
+
+void finalizePR(String prLink) {
+    withCredentials([string(credentialsId: "${GITHUB_TOKEN_CREDS_ID}", variable: 'GITHUB_TOKEN')]) {
+        String currentBodyJson = sh(returnStdout: true, script: "gh pr view ${prLink} --json body").trim()
+        echo "currentBodyjson = ${currentBodyJson}"
+        String newBody = """
+${readJSON(text: currentBodyJson).body}
+${getPRLinksSummary()}
+    """
```
Comment (Ginxo) [inline .ci/jenkins/Jenkinsfile.ecosystem.update-quarkus-all]:
```suggestion
""" 
```

**Review by Ginxo — APPROVED** [review]
LGTM, just few minor suggestions

**Review by rsynek — APPROVED** [review]

**Comment by radtriste** [comment]
Example PR: https://github.com/radtriste/drools/pull/19

**Pull request merged**